linode/manager
# Change: Intercept Unauthed Migrate Errors

## Description

If users are blacklisted from migrating a Linode, we need to intercept the error and prompt the user to open a support ticket.

## Type of Change
- Non breaking change ('update', 'change')

If the any above types of change apply to this pull request, please ensure to include one of the listed keywords in the pull request title.

## To Test

1. Add `dc_migrations_disabled` tag to your account
2. Try to init a migration
3. See error

## Note to Reviewers

Adds a new API for intercepting errors. Basically, pass your conditions for intercepting an error and the new text you want to the `interceptErrors` function like so:

```js
interceptErrors(
   errors, 
   [{ 
       condition: (error: APIError) => error.reason === 'something', 
       replacementText: 'Some new text'
   }]
)
```
